samples/star-wars-api/HttpHandlers.fs

```diff
@@ -17,9 +18,11 @@ type private GQLRequestContent =
       // TODO: Add OperationName handling
       OperationName : string voption
       Variables : ImmutableDictionary<string, JsonElement> voption }
-
+      
```

Comment:
Use https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer to avoid this